### DISCUSSION TITLE
Update readme for new PAT system of Modrinth

### DISCUSSION THREAD

**Discussion by Andre601** [Kira-NT/mc-publish#88] [Ideas]
Modrinth updated their backend which comes with some big changes to their PAT system.

You can now create PATs with fine-grained controls over what permissions it should have (See screenshot below).
The action's readme should mention what scopes are required for it to work (I assume `Create versions` is one), so that people aren't confused when making a new PAT to use for this Action.

![grafik](https://github.com/Kir-Antipov/mc-publish/assets/11576465/bcabb9eb-100d-4827-84e5-c7bca54278c4)

**Comment by Kira-NT** [comment]
The README is already massive, so I would love to reduce its size, not expand it further. I am currently working on creating a standalone documentation for the project where I can provide detailed information on every aspect, including instructions on how to obtain the required tokens.

At the moment, here is the complete list of permissions that `mc-publish` **may** require:

- `Read projects` - this permission is necessary to filter out unsupported loaders for the given project
- `Write projects` - this permission will be required when I implement #64
- `Read versions` - this permission is essential for the `unfeature` functionality to work
- `Write versions` - this permission is also needed for the `unfeature` functionality to function correctly
- `Create versions` - well, this is the whole purpose of `mc-publish`
